lifecycle/lifecycle-common/build.gradle

```diff
@@ -71,7 +71,7 @@ kotlin {
             dependencies {
                 api(libs.kotlinStdlib)
                 api(libs.kotlinCoroutinesCore)
-                api(project(":annotation:annotation"))
```

Comment:
Does annotation support all our platforms? It seems as invalid change

Reply:
Fixed